2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,12 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  let missItems = recipe.ingredients.filter(
+    (item) =>
+      pantry.fridgeContents.indexOf(item) === -1 &&
+      pantry.cupboardContents.indexOf(item) === -1
```

Comment:
you can merge these 2 arrays together and just check that array (array which includes all items in pantry.fridgeCiontent and pantry.cupboardContent)